templates/zerver/help/create-a-stream.md

```diff
@@ -0,0 +1,40 @@
+# Create a stream
+On Zulip, streams are similar to conversation threads or subjects. Streams are located on the left sidebar.
+Streams can be either:
+* **Public streams** - Public streams are for open discussions. All users can subscribe to public streams and discuss there.
+* **Private streams** - Private streams are for confidential discussions and are only visible to users who've been invited to subscribe to it.
+
+## Create a new stream
+To create a stream, locate the cog (![cog](/static/images/help/cog.png)) next to **Streams** in your left sidebar. A tooltip labeled **Subscribe, add, or configure streams** will appear.
+
+![Streams Cog](/static/images/help/streams-1.png)
+
+When you click the cog (![cog](/static/images/help/cog.png)), you'll see an overview of all the streams you've subscribed to. Click the button labeled **Create new stream** to create a stream.
+
+![Streams Overview](/static/images/help/streams-overview.png)
```

Comment:
This is up to you, but it might be nice to graphically highlight the `Create new stream` button somehow in this screenshot.